src/utils/index.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+/**
+ * HTML element attributes.
+ */
+export
+interface IElementOptions {
+  /**
+   * A space-delimited list of classes for the element.
+   */
+   className?: string;
+
+   /**
+    * The title attribute of the element.
+    */
+   title?: string;
+
+   /**
+    * The value attribute of the element.
+    */
+   value?: string;
+}
+
+/**
+ * Create an HTML element with given attributes and children.
+ * 
+ * #### Notes
+ * As a convenience, the children can be a list of elements
+ * or a string (which is interpreted as the textContent of the
+ * new element).
+ */
+
+export
+function createElement(tag: string, attr?: IElementOptions, children?: HTMLElement[] | string): HTMLElement {
```

Comment:
I think I prefer `children` over `content`, since the majority of the time I see it being used to add a list of children. The ability to add a text node is just a convenience. Regardless, the name choice does not change the API because javascript doesn't have keyword arguments.